packages/graphql-config/src/schema.ts

```diff
@@ -1,17 +1,18 @@
-import { DocumentNode, Kind } from "graphql";
+import { parse } from "graphql";
+import { mergeTypeDefs } from "@graphql-tools/merge";
 import CommonSchema from "./schema/common.graphql";
 import ViewDirectiveSchema from "./schema/view.graphql";
 import FormDirectiveSchema from "./schema/form.graphql";
 import ConstraintSchema from "./schema/constraint.graphql";
 
-const mergeDocumentNodes = (docs: DocumentNode[]) => ({
-  kind: Kind.DOCUMENT,
-  definitions: docs.flatMap((doc) => doc.definitions),
-});
```

Comment:
Found that merging like this is not working well, so now here is using use `@graphql-tools/merge` instead.